ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -31,6 +31,7 @@ function parseSidePanelParam(param: string) {
 interface WorkflowDetailsState {
     workflow: Workflow;
     links: Link[];
+    followLogStream: boolean;
```

Comment:
In fact, I'm not sure that the `WorkflowDetails` component needs to keep track of this state at all. All of this should be local to `WorkflowLogsViewer`